src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -663,7 +663,7 @@ String getFilePath(Calendar calendar, String clientPrefix) {
     int day = calendar.get(Calendar.DAY_OF_MONTH);
     int hour = calendar.get(Calendar.HOUR_OF_DAY);
     int minute = calendar.get(Calendar.MINUTE);
-    long time = calendar.getTimeInMillis();
+    long time = calendar.getTimeInMillis() / 1000;
```

Comment:
Thanks fort he fix @sfc-gh-kkloudas !
I would suggest `timeInSeconds` or `seconds = calendar.get(Calendar.SECOND);`

Reply:
+1, but I would suggest we do the same thing as what the server side is doing in case we run into a similar rounding issue

Reply:
The server side is XP.

Reply:
@sfc-gh-gdoci `calendar.get(Calendar.SECOND)` will just give you the seconds in the minute. From the docs: `E.g., at 10:04:15.250 PM the SECOND is 15.`

Reply:
Then let's use `TimeUnit` to convert?